themes/custom/essex/css/overrides.css

```diff
@@ -1439,3 +1439,10 @@ footer .lgd-footer__footer a {
 .lgd-region--content .paragraph--type--localgov-contact a {
   word-break: break-word;
 }
+
+p {
```

Comment:
Will this apply globally and is that ok?

Reply:
Moved into the News component